tests/KernelFlexibleCenterer_tests.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+from skcosmo.preprocessing.FlexibleScaler import KernelFlexibleCenterer
+import sklearn as sk
```

Comment:
Generally you'll see `sklearn` listed as `sklearn` or individual modules imported. I've never seen it referred to as `sk`